tests/providers/amazon/aws/operators/test_batch.py

```diff
@@ -49,7 +48,7 @@ class TestBatchOperator(unittest.TestCase):
     @mock.patch.dict("os.environ", AWS_ACCESS_KEY_ID=AWS_ACCESS_KEY_ID)
     @mock.patch.dict("os.environ", AWS_SECRET_ACCESS_KEY=AWS_SECRET_ACCESS_KEY)
     @mock.patch("airflow.providers.amazon.aws.hooks.batch_client.AwsBaseHook.get_client_type")
-    def setUp(self, get_client_type_mock):
+    def setup(self, get_client_type_mock):
```

Comment:
This will fail, if you decorate setup_method/teardown_method or entire class you need to provide `method` argument see discussions:
- https://github.com/apache/airflow/pull/28039#discussion_r1037813189
- https://github.com/apache/airflow/pull/28039#discussion_r1037814650

```suggestion
    def setup(self, method, get_client_type_mock):
```